mycity/mycity/intents/user_address_intent.py

```diff
@@ -0,0 +1,109 @@
+"""
+Functions for setting and getting the current user address
+"""
+
+from . import intent_constants
+
+
+def set_address_in_session(mcd):
+    """
+    Adds an address from the set address intent to the provided session object

```

Comment:
We could really call this from anywhere, not just the set address intent

Reply:
Updated the function comment.